node/src/BaseClient.ts

```diff
@@ -297,6 +297,35 @@ export type DecoderOption = {
     decoder?: Decoder;
 };
 
+/** A replacement for `Record<GlideString, T>` - array of key-value pairs. */
+export type GlideRecord<T> = { key: GlideString; value: T }[];
+
+// GLideRecord for score for sorted set commands?
+// TODO export
+export type ElementScoreData = { element: GlideString; score: number }[];
```

Comment:
yes, I agree. This way you explicitly convey that this data type is meant for storing elements and their associated scores. This is more explicit than using GlideRecord<number> , where the meaning of "key" and "value" might be less obvious without additional comments